templates/shortcode-profile-editor.php

```diff
@@ -15,18 +15,15 @@
 	$last_name    = get_user_meta( $user_id, 'last_name', true );
 	$last_name    = get_user_meta( $user_id, 'last_name', true );
 	$display_name = $current_user->display_name;
-	$address      = give_get_donor_address( $user_id, array( 'address_type' => 'personal' ) );
-
 	$donor        = new Give_Donor( $user_id, true );
+	$address      = $donor->get_donor_address( array( 'address_type' => 'personal' ) );
 	$company_name = $donor->get_meta( '_give_donor_company', true );
 
 	if ( isset( $_GET['updated'] ) && 'true' === $_GET['updated'] && ! give_get_errors() ) :
 		if ( isset( $_GET['update_code'] ) ) :?>
 			<?php
-			switch ( $_GET['update_code'] ) {
-				case '1':
-					printf( '<p class="give_success"><strong>%1$s</strong> %2$s</p>', esc_html__( 'Success:', 'give' ), esc_html__( 'Your profile has been updated.', 'give' ) );
-					break;
+			if ( 1 === absint( $_GET['update_code'] ) ) {
+				printf( '<p class="give_success"><strong>%1$s</strong> %2$s</p>', esc_html__( 'Success:', 'give' ), esc_html__( 'Your profile has been updated.', 'give' ) );
 			}
 			?>
 			</p>
```

Comment:
@raftaar1191 please check why we have extra `p` here

Reply:
Thanks, @ravinderk remove the extra `P` tag here